configure.ac

```diff
@@ -1204,6 +1204,23 @@ AC_ARG_WITH(
     ]
 )
 
+AC_SUBST(systemd_unit_dir)
+AC_ARG_WITH(
+    systemd-unit-dir,
+    AS_HELP_STRING(
+	    [--with-systemd-unit-dir=DIR],
+	    [systemd unit directory @<:@SYSTEMD_UNIT_DIR@:>@
+        Defaults to /etc/systemd/system/]
```

Comment:
"Defaults to /etc/systemd/system/" is not entirely accurate any more (on non-SUSE anyway)

We can just remove this sentence for simplicity, or explain that the value comes from `pkg-config`